lib/node_modules/@stdlib/blas/ext/base/dsnansumpw/docs/repl.txt

```diff
@@ -1,11 +1,11 @@
 
-{{alias}}( N, x, stride )
+{{alias}}( N, x, strideX )
     Computes the sum of single-precision floating-point strided array elements,
-    ignoring `NaN` values, using pairwise summation with extended accumulation,
-    and returning an extended precision result.
+    ignoring `NaN` values, using pairwise summation with extended
+    accumulation, and returning an extended precision result.
```

Comment:
@headlessNode Would you mind checking line wrapping again? To my eye, this change should be reverted.